client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideCoverPhotoPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  return (
+    <div className="container">
+      <a className={ scss.backLink }
```

Comment:
I think links are one thing that definitely should be their own component since they will have common functionality throughout the app, and will integrate to the global routing functionality, however that is implemented.